extensions/wrapper/wrapper/src/main/java/de/sovity/edc/ext/wrapper/api/offering/OfferingResource.java

```diff
@@ -0,0 +1,50 @@
+package de.sovity.edc.ext.wrapper.api.offering;
+
+import de.sovity.edc.ext.wrapper.api.offering.model.CreateOfferingDto;
+import de.sovity.edc.ext.wrapper.api.offering.services.OfferingService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.Valid;
+import jakarta.ws.rs.Consumes;
+import jakarta.ws.rs.POST;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
+import jakarta.ws.rs.core.Response.Status;
+import lombok.RequiredArgsConstructor;
+
+/**
+ * Provides the endpoints for managing offerings.
+ *
+ * @author Ronja Quensel (ronja.quensel@isst.fraunhofer.de)
+ */
+@Path("wrapper/offering")
+@Tag(name = "Offering", description = "EDC Contract Offering API Endpoints")
```

Comment:
Careful, this creates an OfferingApi not mentioned in EdcClient classes.

We should merge this with the UseCaseApi, as this is the prime endpoint of the use case api. feel free to otherwise clean up the use case api if anything does not seem to fit in there.